py-bindings/setup.py

```diff
@@ -49,6 +49,7 @@ def build_extension(self, ext: CMakeExtension) -> None:
         # EXAMPLE_VERSION_INFO shows you how to pass a value into the C++ code
         # from Python.
         cmake_args = [
+            "-DCMAKE_CXX_COMPILER=/usr/bin/clang++",  # Force Clang for castxml
```

Comment:
Probably want to have this be a variable with the found clang executable, not explicitly hardcoding this path.

Reply:
Maybe move the below clang search to above?

Reply:
Now that we build the macos/arm64 target natively instead of cross-compiling, it seem like we can use the system clang installation instead of the one from homebrew. The clang binary search logic is now gone in https://github.com/ompl/ompl/pull/1188/commits/c817f3668f38cd36a7184a63ee1520453b990740.

I've replaced this bit with a `shutil.where("clang++")` in https://github.com/ompl/ompl/pull/1188/commits/8740aa897b2d0943c9a3e4458d69077aa46e5f2d.